### ISSUE TITLE
Example vimrc and document or two?

### ISSUE THREAD

**Issue by anthrolisp** [Hipomenes/vim-ide#1]
This looks awesome! Thank you!
Any chance that you could include an example vimrc and document or two to help people get up and going?
Many thanks in advance!

**Comment by anthrolisp** [comment]
I'm curious to see the zettelkasten setup in action!

**Comment by ghost** [comment]
Yeah would be nice to see it in action! I'm new in the command line world and tutorials helps a lot to learn the basics
